src/engine-scripts/puppet/scroll.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (page) => {
+	page.evaluate( () => {
```

Comment:
Does this need await?

Reply:
I don't know why but i was getting a timeout with this, presumably because it never resolved

Reply:
It happens infrequently but definitely happens for me:
<img width="1428" alt="Screen Shot 2022-04-18 at 2 57 13 PM" src="https://user-images.githubusercontent.com/3496279/163877444-d6fc1c4f-e67d-4250-a528-150d4e7193c3.png">

Not sure why this is yet as the adding async/await didn't seem to matter.

The other issue I'm seeing is the sticky header tests aren't showing the sticky header for me:
![MediaWiki_Test_sticky_header_logged-in_scroll_0_viewport_1_tablet](https://user-images.githubusercontent.com/3496279/163877456-7e2951f3-3115-4d27-8718-88bc9df2e233.png)

Reply:
Okay, FINALLY worked this one out. I needed to return true in page.evaluate

Pushed new patch.